pkg/proto/readme.md

```diff
@@ -0,0 +1,8 @@
+Notes for protobuf
+
+1. Run this command to compile .proto to .pb.go file.
+```/AgentBaker/hack/tools/bin/buf generate -o .```
```

Comment:
nit: add a target to root-level Makfile instead?

Reply:
Updated the makefile. Do you mean that?